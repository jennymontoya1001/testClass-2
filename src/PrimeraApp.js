import React from 'react'

export const PrimeraApp = ({saludo, subtitulo}) => {
    return (
        <div>
            <h1>{saludo}</h1>
            <p>{subtitulo}</p>
        </div>
    )
}
