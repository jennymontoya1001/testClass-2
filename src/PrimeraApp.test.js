import '@testing-library/jest-dom';
import React from "react";
import {shallow} from 'enzyme';
import { PrimeraApp } from "./PrimeraApp";

describe('pruebas <Primera App/>', () => {
    test('validar render de <PrimeraApp/>', () => {
        const saludoL = 'Hola Clase';
        const subtituloL = 'Bienvenidos';

        const wrapper = shallow(<PrimeraApp
         saludo={saludoL}
         subtitulo={subtituloL}
        />)

        expect(wrapper).toMatchSnapshot();
    })
})
