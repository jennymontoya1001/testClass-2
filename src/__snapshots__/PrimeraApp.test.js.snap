// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas <Primera App/> validar render de <PrimeraApp/> 1`] = `
<div>
  <h1>
    Hola Clase
  </h1>
  <p>
    Bienvenidos
  </p>
</div>
`;
